modules/OpenWeather/module.json

```diff
@@ -59,17 +81,35 @@
 			},
 			"units": {
 				"label": "__l_units__",
-				"optionLabels": ["°C", "°F"]
+				"type": "select"
+			},
+			"units_wind": {
+				"type": "select"
```

Comment:
Worth adding optionLabels to allow translations of km/h, mph,... But optional

Reply:
I don't think that's really necessary, as it's understandable...